report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Options.java

```diff
@@ -199,7 +199,7 @@ public enum Locations {
     }
 
     public enum Side {
-        Head, Tail
```

Comment:
Just brainstorming on that:
head/tail x  head/headEach/tail/tailEach x headAll/headEach/tailAll/tailEach  or even headAll/head/tailAll/tail 

The concern is a bit in future - 
 - how it will behave above the three scenarios we drafted?
> 1 - join statusLine and all t.getOutputs() to one huge string,. print one number

Thats  the `headAll/headEach/tailAll/tailEach `

> 2 - comapre adequate get.value() (where adequate is equality of get.getName - not the > order; if no adequate, then 0% match). Status l;ine can be coonsidered as special key.
 >2.1 - make avarage od vlaues, and print one number
> 2.2 - print one N% numbers, each per key comparison.
That is the `head/tail`. 

Whchis is leaving us with current `Head, HeadEach, Tail, TailEach`, where I in 2  `Head, HeadEach` and `Tail, TailEach` will do the same ?

Reply:
I haven't thought about that. This feature was done with only 1 in mind - joining the traces into huge string. So in my mind, it makes sense that with 2 (comparing different outputs), `Head`, `HeadEach` and `Tail`, `TailEach` will do the same thing.